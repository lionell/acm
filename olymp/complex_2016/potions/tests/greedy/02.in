100 4
74 99
53 1
4 82
31 29
68 3
78 38
91 59
2 45
25 44
28 51
16 75
4 53
65 59
67 23
62 76
73 88
86 34
16 25
53 89
89 44
79 22
78 68
11 71
15 77
28 35
8 40
90 41
98 10
32 55
31 63
51 91
25 5
29 35
24 38
80 79
98 12
77 59
30 75
52 87
20 42
82 78
17 27
34 72
28 15
78 65
56 56
58 27
62 78
17 84
64 83
92 96
53 60
35 69
54 83
23 85
61 71
96 5
75 77
90 30
95 1
56 68
92 49
28 15
98 52
36 42
36 62
21 28
95 88
69 55
38 49
19 92
94 98
76 15
44 67
67 66
32 37
86 24
38 24
76 12
51 34
94 10
35 86
72 15
67 30
77 40
30 6
4 95
10 24
89 79
43 51
88 56
77 74
96 72
81 13
47 72
79 32
16 41
53 14
44 85
25 20
