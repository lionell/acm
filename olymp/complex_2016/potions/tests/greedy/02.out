0.0509317